js/interface.js

```diff
@@ -1,78 +1,176 @@
-var $folderContents = $('#folder-contents');
+// VAR SETUP //
+var $folderContents = $('.file-table-body');
+var $folderList;
+var $organizationAppList = $('.dropdown-menu-holder ul');
 var templates = {
   file: template('file'),
-  folder: template('folder')
+  folder: template('folder'),
+  folderItem: template('folder-item'),
+  organizations: template('organizations'),
+  apps: template('apps')
 };
+
 var currentFolderId;
 var currentFolders;
 var currentFiles;
+var counter;
```

Comment:
I was suggesting just to rename it, like `counterOrganisation`. You could also get rid of it and use something like `if ($organizationAppList.find('li').length === 1)`